setup/context/commands_test.go

```diff
@@ -0,0 +1,69 @@
+package context
```

Comment:
Thanks, both. I appreciate the discussion here. Part of why I opted to create the mock class directly was because of my unfamiliarity with gomock, for sure. It also served as a bit of a learning exercise for working with Go. I will certainly look into using the best tools for the project.

I have to differ with the percieved value of the unit tests. Although it is true that the _current_ implementation is essentially just wrapping `json.Marshal`, that could certainly change; e.g., more robust validation against certain schemas could be added or it could be tweaked another way. The unit tests are there as sentinels to ensure that behavior works as expected when things are changed.

Regardless, I agree with the larger points expressed above. The unit tests have been dropped, and the mock class has been removed.